js/graviex.js

```diff
@@ -0,0 +1,991 @@
+        //         "currency": "ltc",
+        //         "amount": "0.14108256",
+        //         "fee": "0.0",
+        //         "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //         "uid": null,
+        //         "created_at": "2022-02-01T18:10:32+03:00",
+        //         "confirmations": "6",
+        //         "done_at": null,
+        //         "state": "accepted"
+        //     }
+        // ]
+        return this.parseTransactions (response, currency, since, limit);
+    }
+
+    parseOrderStatus (status) {
+        const statuses = {
+            'ACTIVE': 'wait',
+            'CANCELED': 'cancel',
+            'FILLED': 'done',
+            'REJECTED': 'rejected',
+            'EXPIRED': 'expired',
+        };
+        return this.safeString (statuses, status, status);
+    }
+
+    parseOrder (order, market = undefined) {
+        //   {
+        //     "id": 287792646,
+        //     "at": 1643956089,
+        //     "side": "sell",
+        //     "ord_type": "limit",
+        //     "price": "0.141",
+        //     "avg_price": "0.0",
+        //     "state": "cancel",
+        //     "market": "ltcbtc",
+        //     "created_at": "2022-02-04T09:28:09+03:00",
+        //     "volume": "0.141",
+        //     "remaining_volume": "0.141",
+        //     "executed_volume": "0.0",
+        //     "trades_count": 0,
+        //     "strategy": null
+        //   }
+        const id = this.safeString (order, 'id');
+        const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
+        const sideType = this.safeString (order, 'ord_type');
+        const side = this.safeString (order, 'side');
+        const price = this.safeString (order, 'price');
+        const amount = this.safeString (order, 'avg_price');
+        const remaining = this.safeString (order, 'remaining_volume');
+        const status = this.parseOrderStatus (this.safeString (order, 'state'));
+        const marketId = this.safeString (order, 'market');
+        market = this.safeMarket (marketId, market, '_');
+        const symbol = market['symbol'];
+        const rawTrades = this.safeValue (order, 'trades', []);
+        return this.safeOrder ({
+            'info': order,
+            'id': id,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'symbol': symbol,
+            'type': sideType,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'remaining': remaining,
+            'status': status,
+            'trades': rawTrades,
+        }, market);
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        if (id === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrder() requires a order id argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+        };
+        if (id !== undefined) {
```

Comment:
this construct is strange, let's remove it

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/pull/11697/commits/162b106bdb42084917d6187a3a60fb6e0f55e7d1